src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_help.py

```diff
@@ -120,7 +120,11 @@
 helps['ams transform update'] = """
     type: command
     short-summary: Update the details of a transform.
-"""
+    examples:
+        - name: Update a transform by setting up a new outputs list with AudioAnalyzer built-in preset and a custom preset from a local JSON file.
```

Comment:
Wouldn't it be 'output list' instead of 'outputs list'?

Reply:
agree with that